client/controllers/searchInput.coffee

```diff
@@ -0,0 +1,58 @@
+###
+ Seach input options/settings:
+    textFilter:  Either a reactiveVar or ReactiveTableFilter
+    id:          The id associated with the element and ReactiveTableFilter if its passed
+    props:       If a textFilter is null, a ReactiveTableFilter will be created
+                 with the id and props (an array)
+    toggleable:  If set to true, the input will be initially hidden and appear when
+                 the search icon is clicked. When the user clicks on the search
+                 icon within the input, the input returns to its hidden state.
+    placeholder: Defaults to 'Search'
+    classes:     Classes which will be applied to the input's parent element
+####
+
+clearSearch = (instance) ->
+  instance.textFilter.set('')
+  instance.$('.search').val('')
+
+Template.searchInput.onCreated ->
+  instanceData = @data
+  searching = true
```

Comment:
hrrm - yeah I didn't think of that case.  I'd probably just leave it the way it is then.